readme.md

```diff
@@ -837,6 +837,32 @@ Accepts the same values as [`color`](#color) in `<Text>` component.
 
 <img src="media/box-borderColor.jpg" width="228">
 
+##### Measuring dimensions
+
+Use `measureElement()` function if you need to measure dimensions of a particular `<Box>` element.
+It returns an object with `width` and `height` properties.
+This function is useful when your component needs to know the amount of available space it has before it can start rendering or when you need to change your layout based on the length of the content.
+
+**Note:** `measureElement()` returns correct results after initial render, when layout has been calculated.
```

Comment:
The clear enough what implications this has for usage. What does it return before? How should the user handle this?